cogs/onmyoji.py

```diff
@@ -95,6 +101,59 @@ def generate_user_shikigami_locations(cls, shiki_name):
                 user_database_filtered_locations.append(temp_list)
         return user_database_filtered_locations
 
+class Embeds:
+    def shard_trading_embed(self, user):
+        need_list = ''
+        have_list = ''
+        for i in self.shard_trading_db[str(user.id)]['need']:
```

Comment:
```suggestion
        need_list = '\n'.join([bold_shard_text(i) for i in self.shard_trading_db[str(user.id)]['need']]
```
Then just make a bold_shard_text function

Reply:
Also you don't want to use i unless its an `int` (cause standards) `arg` might be better?